### PR TITLE
Feature/article page title

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[維運] 滿版版型標題調整](https://app.asana.com/0/1203699264568808/1206117666290873/f)

# Notice

# Dependency

### CODE REVIEW

File: packages/core/src/constants/z-index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  hamburgerMenu: 4000,
+  toc: 3000, // table of contents
+  webPush: 2000,
+  buttomBanner: 2000,
+  header: 1000,
+  articleFullScreenHeroImage: 900,
+  embedUp: 800,
+  mobileToolBar: 700,
+  embedDown: 600,
+  embed: 500,
```
Comment (Aylie-Chou) [inline packages/core/src/constants/z-index.js:13]: Question: where do we use `embed` value ?

Comment (liruchen32) [reply packages/core/src/constants/z-index.js:13]: image

Comment (liruchen32) [reply packages/core/src/constants/z-index.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]
ＬＧＴＭ

**Pull request merged**